### PR TITLE
Further 1825 assets added

### PR DESCRIPTION
**Pull request by thepackrat**
This is part of getting 1825 Unit 3 into play. TODO list added at the top of the file.

### CODE REVIEW

File: lib/engine/game/g_1825/game.rb [hunk 1]
```diff
@@ -91,181 +135,73 @@ class Game < Game::Base
...
+            value: 30,
+            revenue: 5,
+            color: :Green,
+            abilities: [{ type: 'no_buy' }],
           },
           {
-            name: 'Ostbayrische Bahn',
-            sym: 'OBB',
-            value: 120,
+            name: 'Tanfiedld Wagon Way',
```
Comment (jenf) [inline lib/engine/game/g_1825/game.rb:191]: Tanfield

File: lib/engine/game/g_1825/game.rb [hunk 2]
```diff
@@ -332,6 +265,9 @@ class Game < Game::Base
           },
         ].freeze
 
+        # MINORS = [
```
Comment (jenf) [inline lib/engine/game/g_1825/game.rb:268]: Remove commented out code

Comment (thepackrat) [reply lib/engine/game/g_1825/game.rb]: Was going back and forth how best to implement these ones. Done.

File: lib/engine/game/g_1825/game.rb (hunk 1 above)
Comment (thepackrat) [reply lib/engine/game/g_1825/game.rb]: fixed.

**Review by tobymao — APPROVED** [review]

**Pull request merged**